File: AACommoditiesDashboardUi/src/app/models/commodity-metrics.ts


export class CommodityMetrics {
    commodityId?: number;
    commodity?: string;
    price?: number;
    position?: number;
    pnlDaily?: number;
    pnlYTD?:number;


    constructor(commodityId: number, commodity: string,price: number, position: number, pnlDaily: number, pnlYTD:number) {
        this.commodityId=commodityId;
        this.commodity=commodity;
        this.price=price;
        this.position=position;
        this.pnlDaily=pnlDaily;
        this.pnlYTD=pnlYTD;
      }
  }

File: AA.CommoditiesDashboard.Api/AA.CommoditiesDashboard.Api/Model/CommodityMetrics.cs
using System;

namespace AA.CommoditiesDashboard.Api.Model
{
    public class CommodityMetrics
    {
        public int CommodityId { get; set; }
        public string Commodity { get; set; }

        public decimal Price { get; set; }

        public int Position { get; set; }

        public decimal PnlDaily { get; set; }
        
        public decimal PnlYTD { get; set; }
    }
}